packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -230,33 +243,37 @@ export const useSliderStyles_unstable = (state: SliderState): SliderState => {
   const railStyles = useRailStyles();
   const thumbStyles = useThumbStyles();
   const inputStyles = useInputStyles();
+  const isVertical = state.vertical;
 
   state.root.className = mergeClasses(
-    sliderClassName,
+    sliderClassNames.root,
```

Comment:
you should apply the root clasname last since you want user overrides to win over internal styles